adanet-co2-monitor.ino

```diff
@@ -271,7 +281,8 @@ void setup()
   const int16_t footerY = display.height() - 1 - FOOTER_SIZE * CHAR_HEIGHT;
   if (error == ERROR_NONE)
   {
-    printfAligned(HEADER_SIZE, ALIGN_LEFT, headerY, EPD_BLACK, "% 3.1f%cC", temperature, 0xF7);
+    float disp_temp = (temp_units == 'C') ? temperature : (temperature / 5.f * 9.f + 32.f);
+    printfAligned(HEADER_SIZE, ALIGN_LEFT, headerY, EPD_BLACK, "% 3.1f%c%c", disp_temp, 0xF7, temp_units);
```

Comment:
Will this fit in `% 3.1f` if temp (in F) is above 100? Might need to use `% 3.0f` for freedom units?

Reply:
Wait, `% 3.1f` is wrong in the first place--should be `% 5.1f` for C (since the total length accounts for the space/negative sign and decimal point), assuming we'll never see triple-digit C

Reply:
Yes, %5.1 makes sense. It will also work for F since we'll never see negative triple digit temps in F.

Reply:
We might if someone has the monitor outdoors in the summer?

Reply:
I'm confused, negative triple digits in the summer?

With %5.1 we're good with triple digits in the summer, say 120.9F. 
With %5.1 we're also good with double digits and a negative sign in the winter, say -45.9F

Reply:
Ohh ok `%5.1f`, not `% 5.1f`. Yup, agreed, that should be fine.